### ISSUE TITLE
no bower.json file

### ISSUE THREAD

**Issue by ivanparvaresh** [intuxicated/css-persian#2]
hi, 
we will be so happy to have bower package
i can do it if you like :)
thanks a lot

**Comment by Bersam** [comment]
yeah it would be great if you add a bower.json file to this repo.
